src/components/SliderBox/slidesData.ts

```diff
@@ -0,0 +1,19 @@
+type slidesArrayType = {
```

Comment:
use interface and provide a more convenient name. Use capital letter

Reply:
fixed